package.json

```diff
@@ -79,6 +79,7 @@
     "got": "^10.7.0",
     "minimist": "^1.2.0",
     "ora": "^5.4.0",
+    "path-key": "^3.1.1",
```

Comment:
`cross-spawn` already depends on this module at the same version, so this is only making a sub-dependency a “real” dependency.